Corona-Warn-App/src/main/res/navigation/trace_location_attendee_nav_graph.xml

```diff
@@ -11,7 +11,7 @@
         tools:layout="@layout/fragment_confirm_check_in">
         <argument
             android:name="traceLocation"
-            app:argType="de.rki.coronawarnapp.ui.eventregistration.attendee.checkins.VerifiedTraceLocation" />
+            app:argType="de.rki.coronawarnapp.eventregistration.events.TraceLocation" />
```

Comment:
This it is used in UI I would keep it in UI` package `. this is the main reason why it is Praceable in the first place is not it ?

Reply:
TraceLocation is not a "UI only" class, it's being used by the repository and I think also when verifying the server signed QR Code.

Reply:
I also see it as a "core" class.

Reply:
Ok, I think discussion regarding that can go forever 😄. I think using it for the server does not require to have it `Pracelable` does it ?.  if you both agree why not. democracy wins